### PR TITLE
Assets

### PR DESCRIPTION
**Pull request by IlyaKirsanov**

### CODE REVIEW

File: src/components/Modal/GetInTouch/index.js [hunk 1]
```diff
@@ -250,7 +249,7 @@ export default class GetInTouch extends PureComponent {
         <div className={`new-black-bg ${showClass && 'block-animation'}`}></div>
         <div className={`modal-wrapper ${showClass}`}>
           <span className={`close-btn-x  ${showClass}`} onClick={this.close}>
-            <img src={closeImgX} alt="close btn" />
+            <img src="https://phobosys-assets.s3.us-east-2.amazonaws.com/img/close-btn-x.png" alt="close btn" />
```
Comment (sergiylegmelnik) [inline src/components/Modal/GetInTouch/index.js:252]: You should use CloudFront link instead.

**Review by sergiylegmelnik — CHANGES_REQUESTED** [review]

File: src/images&video/index.js [hunk 2]
```diff
@@ -75,15 +44,27 @@ export const newsImages = {
...
-allocine,
-beeline,
-dreamHost,
-earnin,
-genius,
-okta,
-nice,
-roblox,
-flyfirst,
+  allegion:'https://phobosys-assets.s3.us-east-2.amazonaws.com/svg/allegion.svg' ,
```
Comment (sergiylegmelnik) [inline src/images&video/index.js:60]: Move base link to the configuration file.

**Review by sergiylegmelnik — CHANGES_REQUESTED** [review]

File: src/images&video/index.js (hunk 2 above)
Comment (IlyaKirsanov) [reply src/images&video/index.js:60]: please clerify, what exactly configuration file I should use for this base link

File: src/components/Modal/GetInTouch/index.js (hunk 1 above)
Comment (IlyaKirsanov) [reply src/components/Modal/GetInTouch/index.js:252]: done

**Review by sergiylegmelnik — APPROVED** [review]

**Pull request merged**